src/jvc/src/plugin/AntiCensorPlugin.js

```diff
@@ -57,7 +57,7 @@ export class AntiCensorPlugin {
                 for (const key in AntiCensorPlugin.associations) {
                     const value = AntiCensorPlugin.associations[key];
                     const [from, to] = [key, value];
-                    const regex = new RegExp(from, 'gi');
+                    const regex = new RegExp(`\\b[.,;!?()'"-]?${from}[.,;!?()'"-]?\\b`, 'gi');
```

Comment:
Je ne vois pas l'intérêt d'ajouter `[.,;!?()'"-]?` en début et fin de regex, tu as un exemple ou ça marche pas sans ? Car de mon côte la Regex fonctionne très bien comme ça:
```suggestion
                    const regex = new RegExp(`\\b${from}\\b`, 'gi');
```

Reply:
Exemples;
```javascript
> 'Bonjour orange'.replace(/\borange\b/gi, "CENSORED")
'Bonjour CENSORED'
> 'Bonjour (orange)'.replace(/\borange\b/gi, "CENSORED")
'Bonjour (CENSORED)'
> 'Bonjour eloranged'.replace(/\borange\b/gi, "CENSORED")
'Bonjour eloranged'
> 'Bonjour:orange'.replace(/\borange\b/gi, "CENSORED")
'Bonjour:CENSORED'
```